tests/scripts/all.sh

```diff
@@ -1290,6 +1290,38 @@ component_test_no_use_psa_crypto_full_cmake_asan() {
     if_build_succeeded env OPENSSL_CMD="$OPENSSL_NEXT" tests/compat.sh -e '^$' -f 'ARIA\|CHACHA'
 }
 
+component_test_psa_crypto_config_basic() {
```

Comment:
This test was created per the request from the ticket and asked to have this specific configuration
In all.sh, add a build+test with the full config plus MBEDTLS_PSA_CRYPTO_CONFIG and MBEDTLS_PSA_CRYPTO_DRIVERS and PSA_CRYPTO_DRIVER_TEST minus MBEDTLS_USE_PSA_CRYPTO. I believe the idea is that we want to test that the MBEDTLS_PSA_CRYPTO_CONFIG flag works with the drivers while not having the USE_PSA_CRYPTO enabled. I do not fully understand the reasoning on the USE_PSA_CRYPTO being disabled but I believe Gilles had a reason.

Reply:
Enabling `MBEDTLS_USE_PSA_CRYPTO` enables more code where guards on `MBEDTLS_xxx_C` have to change to `PSA_WANT_xxx`. This is phase 1 where making things work with `MBEDTLS_USE_PSA_CRYPTO` is out of scope (it's filed as #3674 and #3677). Since I don't expect `MBEDTLS_USE_PSA_CRYPTO+MBEDTLS_PSA_CRYPTO_CONFIG` to work at this stage, testing of `MBEDTLS_PSA_CRYPTO_CONFIG` should disable `MBEDTLS_USE_PSA_CRYPTO`.

Reply:
@ronald-cron-arm Based on @gilles-peskine-arm and my comments, is there any changes that need to be made anymore. Seems like most of the issues are addressed by the comments.

Reply:
I am fine with the explanations related to MBEDTLS_USE_PSA_CRYPTO. ok to unset it and to keep this test component as it is.

Otherwise, I think there would be some value in adding a component with just ( full + MBEDTLS_PSA_CRYPTO_CONFIG ) and no test driver to check that the library is suitably configure when both psa/crypto_config.h and mbedtls/config.h are in play to configure the library.

Reply:
This issue is addressed in the latest version with the component `component_test_psa_crypto_config_no_driver`.